sympy/solvers/ode/tests/test_ode.py

```diff
@@ -1313,10 +1307,11 @@ def test_classify_sysode():
     -y(t)**5 + Derivative(y(t), t)], 'order': {y(t): 1, x(t): 1}}
     assert classify_sysode(eq11) == sol11
 
-    eq12 = (Eq(x1, y(t)), Eq(y1, x(t)))
-    sol12 = {'no_of_equation': 2, 'func_coeff': {(0, x(t), 0): 0, (1, x(t), 1): 0, (0, x(t), 1): 1, (1, y(t), 0): 0, \
-    (1, x(t), 0): -1, (0, y(t), 1): 0, (0, y(t), 0): -1, (1, y(t), 1): 1}, 'type_of_equation': 'type1', 'func': \
-    [x(t), y(t)], 'is_linear': True, 'eq': [-y(t) + Derivative(x(t), t), -x(t) + Derivative(y(t), t)], 'order': {y(t): 1, x(t): 1}}
+    eq12 = (Eq(x1, y(t)), Eq(y1, t*x(t)))
```

Comment:
Why is this equation changed?

Reply:
Should I remove tests like these? I was not sure before but these equations were modified because the ```classify_sysode``` function gave different results and these kind of tests were supposed to be transferred to ```test_systems.py```

Reply:
I think it's better to leave the ODE as it is but update the expected result in the test